spot-client/src/spot-remote/ui/views/remote-control.js

```diff
@@ -52,7 +51,6 @@ export class RemoteControl extends React.PureComponent {
             <View name = 'remoteControl'>
                 { this._getView() }
                 <ElectronDesktopPickerModal />
-                <ReconnectOverlay />
```

Comment:
With the overlay gone and the loading screen displaying, does the in-meeting reconnect work okay? I can see a time where maybe a prosody deploy has to happen and connection lost momentarily while in meeting.

Reply:
I will check that

Reply:
We could also redirect back home maybe if that's easier somehow.

Reply:
Okay glad you've asked me to check on that, because I've found serious bug with a reconnect loop(see the last squash commit)

Reply:
What happens to the TV when in-meeting and the disconnect occurs and then the connection recovers?

Reply:
So I was testing the case where RCS prosody is restarted to simulate disconnect on the remote control plane. Both TV and remote(a permanent one) will come back to the meeting and it looks good.

Now I will try another case where will just turn to "offline" in the network conditioner so that both RCS prosody and the meeting prosody will disconnect and see what happens.

Reply:
Yes, it's all good on the in-meeting reconnects. TV comes back to the meeting and remote reconnect correctly to the Spot TV.